server/README.md

```diff
@@ -0,0 +1,77 @@
+# API
+<!-- Endpoints are meant to be preceded by /api (/operations == /api/operations)-->
+## Bookings
+- GET `/bookings`
+  - Param: Time frame (start date and end date)
+  - Response: List of all bookings
+  ##### NOTE: This endpoint is intended to be used for statistics 
+
+## Lectures
+- GET `/lectures`
+  - Param: NONE
+  - Response: List of all lectures
+- GET `/lectures/{lecture_id}`
+  - Param: lecture_id
+  - Response: Lecture object for the given id
+- GET `/lectures/{lecture_id}/bookings`
+  - Param: lecture_id
+  - Response: List of all bookings for a given lecture
+- PUT: `/lectures/{lecture_id}`
+  - Param lecture_id
+  - Body: Lecture object (used to possibly cancel the lecture)
+  - Response: NONE
```

Comment:
[Story 9] Maybe a `PATCH` can be useful to only move a lesson from physical to remote.